### PR TITLE
Optimize if statement

### PR DESCRIPTION
**Pull request by gaozhangmin**
### Motivation

Some  `if` statement need optimize, reduce code.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no) no
  - The public API: (yes / no) no
  - The schema: (yes / no / don't know) no
  - The default values of configurations: (yes / no) no
  - The wire protocol: (yes / no) no
  - The rest endpoints: (yes / no) no
  - The admin cli options: (yes / no) no 
  - Anything that affects deployment: (yes / no / don't know) no

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
no new features.
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java [hunk 1]
```diff
@@ -559,15 +556,17 @@ private synchronized void updateRealtimeResourceQuota() {
...
-        boolean needUpdate = true;
-        if (!oldQuota.getDynamic() || (Math
-                .abs(newQuota.getMsgRateIn() - oldQuota.getMsgRateIn()) < RESOURCE_QUOTA_MIN_MSGRATE_IN
-                && Math.abs(newQuota.getMsgRateOut() - oldQuota.getMsgRateOut()) < RESOURCE_QUOTA_MIN_MSGRATE_OUT
-                && Math.abs(newQuota.getBandwidthIn() - oldQuota.getBandwidthOut()) < RESOURCE_QUOTA_MIN_BANDWIDTH_IN
-                && Math.abs(newQuota.getBandwidthOut() - oldQuota.getBandwidthOut()) < RESOURCE_QUOTA_MIN_BANDWIDTH_OUT
-                && Math.abs(newQuota.getMemory() - oldQuota.getMemory()) < RESOURCE_QUOTA_MIN_MEMORY)) {
-            needUpdate = false;
-        }
+        boolean needUpdate = oldQuota.getDynamic() && (!(Math
```
Comment (Jason918) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java:559]: IMHO, the previous code format is better.

**Review by codelipenghui — APPROVED** [review]

**Review by merlimat — COMMENTED** [review]
I think the original forms are more explicit about the intention and more readable in the end.

**Comment by gaozhangmin** [comment]
@merlimat  pTAL again.

**Review by merlimat — APPROVED** [review]

**Pull request merged**